framework/src/org/checkerframework/framework/type/AnnotatedTypeMirror.java

```diff
@@ -2101,6 +2110,36 @@ private AnnotatedIntersectionType(
         }
 
         @Override
+        public Set<AnnotationMirror> getAnnotations() {
+            Set<AnnotationMirror> glbs = null;
```

Comment:
What is the relationship between the value of the `annotations` field and the result of this method?
So far, the two were interchangeable. I'm a bit fearful of removing the `final` on the parent method and allowing such overrides. You changed one use of `annotations` to the method call. Are there others that should be changed?